streams/service/src/main/java/com/hortonworks/streamline/streams/service/SchemaResource.java

```diff
@@ -109,18 +116,23 @@ public Response postStreamsSchema(StreamsSchemaInfo streamsSchemaInfo,
     @Produces(MediaType.APPLICATION_JSON)
     public Response getKafkaSourceSchema(@PathParam("topicName") String topicName,
```

Comment:
This endpoint was probably never used before. I am okay to keep it as it is to return latest version on MASTER but may be we can just get rid of it?